src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -3,7 +3,18 @@
 public class ConcurConstants {
     public static final String AUTHORIZATION_PROPERTY = "Authorization";
     public static final String BASIC_AUTHENTICATION_SCHEME = "Basic";
-    
+    public static final String OAUTH_AUTHENTICATION_SCHEME = "OAuth ";
+
     public static final int VALIDATION_RESULT_MESSAGE_MAX_LENGTH = 2000;
 
+    public static final String REFRESH_TOKEN_URL_PARAM = "refresh_token=";
+    public static final String CLIENT_ID_URL_PARAM = "&client_id=";
+    public static final String CLIENT_SECRET_URL_PARAM = "&client_secret=";
```

Comment:
this is probably ok, but usually, we would set constants like this to just the string (ex: "refresh_token") and then concatenate them with more global constants for equals (=) and ampersand (&)